bucket/xnresize.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "1.00",
+    "description": "A fast and powerful image resizer",
+    "homepage": "https://www.xnview.com/en/xnresize/",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.xnview.com/XnResize-win-x64.zip",
+            "hash": "472bd46899743f7f6cb602fc09c92185ca3b287a9841a152e545174dc0cd78a7"
+        }
+    },
+    "pre_install": [
+        "if ((Test-Path \"$env:APPDATA\\XnResize\\xnresize.ini\")) { Move-Item -Force -Path \"$env:APPDATA\\XnResize\" -Destination \"$dir\"}",
+        "elseif (Test-Path \"$persist_dir\\xnresize.ini\") { Copy-Item \"$persist_dir\\xnresize.ini\" \"$dir\" -Force }",
+        "else { New-Item -Path \"$dir\\xnresize.ini\" -ItemType 'file' | Out-Null }"
+    ],
```

You do not need appdata. If the file is created next to executable it is used....

Move pre_install after extract_dir..

```suggestion
    "pre_install": [
        "if (-not (Test-Path \"$persist_dir\\xnresize.ini\")) {",
        "    New-Item \"$persist_dir\" -ItemType Directory -ErrorAction SilentlyContinue -Force | Out-Null",
        "    Set-Content \"$dir\\xnresize.ini\" '<FILL SOME BATCH, which i cannot remember>', 'update=0' -Encoding Ascii",
        "} else {",
        "    Copy-Item \"$persist_dir\\xnresize.ini\" \"$dir\"",
        "}"
    ],
```